prow.go

```diff
@@ -304,6 +310,8 @@ func (m *jobManager) newJob(job *Job) error {
 			}
 			args = append(args, fmt.Sprintf(`--namespace=$(NAMESPACE)`))
 
+			script := fmt.Sprintf(scriptTemplate, configEnvVar, configEnvVar, configEnvVar, configEnvVar)
```

Comment:
This makes me uncomfortable because it's such a big change.  Can we cleanly change it in the script without templating?

Reply:
Can we just append something to the front of the script that sets the env var name correctly.

Reply:
Reworked this to avoid envvar and bash templating.
Script is being assembled from:
* `scriptPre`
* instead of `CONFIG_SPEC_n` / `JOB_SPEC_n` a section of `scriptChild` is being templated
* `scriptPost` is appended

That makes us move `container.Command` part below `job.Inputs` parsing though